src/app/models/user.ts

```diff
@@ -1,4 +1,7 @@
 export class User {
+    user_id: number;
     username: string;
     password: string;
+    confirmPassword: string;
+    role_id: number;
```

Comment:
roleId camelCase?

Reply:
can't change to camelCase because in user creation backend need `role_id` and we are passing user as a model.